internal/test/smoke/rbac_permission_test.go

```diff
@@ -109,20 +114,25 @@ func (s *SuiteRbacPermission) doTestDomainList(t *testing.T) int {
 	return res.StatusCode
 }
 
-func (s *SuiteRbacPermission) commonRun(profile string, testCases []TestCasePermission) {
+func (s *SuiteRbacPermission) commonRun(rbacProfile RBACProfile, testCases []TestCasePermission) {
 	t := s.T()
 
+	xrhidProfiles := []XRHIDProfile{XRHIDUser}
```

Comment:
Is this line missing a service account (`xrhidProfiles := []XRHIDProfile{XRHIDUser, XRHIDServiceAccount}`) or is that added in the following PRs - I didn't read them yet?

(otherwise the new  `for` doesn't make sense)

Reply:
Answering to myself: yes it is added in next pr: https://github.com/podengo-project/idmsvc-backend/pull/233/commits/d7d1c3d6110e8f479fea77eb8e0cf02d0ab70b49#diff-c2ee44974ea374e676d4bfc79c174c3dc367af8da4b273e2f4a564528030e61d